$NetBSD: patch-Cargo.toml,v 1.1 2022/01/03 12:19:38 pin Exp $

Don't pull gmp-mpfr-sys from git-HEAD
Wait for a proper release

--- Cargo.toml.orig	2022-01-01 16:02:11.000000000 +0000
+++ Cargo.toml
@@ -1,6 +1,3 @@
-[patch.crates-io]
-gmp-mpfr-sys = { git = "https://gitlab.com/tspiteri/gmp-mpfr-sys.git" }
-
 [workspace]
 members = ["kalk", "cli"]
 
